app/modules/settings/actions/settings-login-user.js

```diff
@@ -0,0 +1,15 @@
+import { settingsUpdateUserState } from "./settings-update-user-state"
+
+const ipcRenderer = require("electron").ipcRenderer
```

Comment:
:art: `import { ipcRenderer }  from "electron"`

Reply:
This was actually on purpose because I'm stubbing the ipcRenderer in the test and I wanted to get the same reference. For some reason it wasn't working with the `import` syntax, will revisit this in the future to see if there's a better way to stub in the tests.